modelina-website/src/components/playground/options/CSharpGeneratorOptions.tsx

```diff
@@ -113,6 +120,20 @@ class CSharpGeneratorOptions extends React.Component<
             />
           </label>
         </li>
+        <li>
+          <label className="flex items-center py-2 justify-between cursor-pointer">
+            <span className="mt-1 max-w-2xl text-sm text-gray-500">
+              Include OverWrite Equal Support
+            </span>
+            <input
+              type="checkbox"
+              className="form-checkbox cursor-pointer"
+              name="csharpOverwriteEqual"
+              checked={this.context?.csharpOverwriteEqual}
+              onChange={this.onChangeOverwriteEqualSupport}
```

Comment:
We can have the `onChange` handler function called `onChangeOverwriteEqual`.
Any reason we're calling it `onChangeOverwriteEqualSupport` ?

Reply:
Sorry, let it be it's fine.